recipes/freetds/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = 'freetds' %}
+{% set version = "1.00.44" %}
+{% set sha256 = "b4376e685e6e133e5c4dc9b1099ddfc6cd264f5313ad5e4c26879f7ae9276194" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: freetds-{{ version }}.tar.gz
+  url: ftp://ftp.freetds.org/pub/freetds/stable/freetds-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and py >= 35]
```

Comment:
Even though we are not building `vc10` (Python 3.4) it would be nice to have it in the recipe so other can use conda-forge recipes locally.